README.md

```diff
@@ -395,24 +406,28 @@ GTMOAuth2 keychain, and the new details for all other GTMAppAuth calls.
 Keychain migration example:
 
 ```objc
+// Create a keychain store
+GTMKeychainStore keychainStore = [[GTMKeychainStore alloc] initWithItemName:kNewKeychainName];
+
 // Attempt to deserialize from Keychain in GTMAppAuth format.
-id<GTMFetcherAuthorizationProtocol> authorization =
-    [GTMAppAuthFetcherAuthorization authorizationFromKeychainForName:kNewKeychainName];
+NSError *error;
+GTMAuthSesion *authSession =
+    [keychainStore retrieveAuthSessionAndReturnError:&error];
 
 // If no data found in the new format, try to deserialize data from GTMOAuth2
-if (!authorization) {
+if (!authSession) {
   // Tries to load the data serialized by GTMOAuth2 using old keychain name.
   // If you created a new client id, be sure to use the *previous* client id and secret here.
-  authorization =
-      [GTMOAuth2KeychainCompatibility authForGoogleFromKeychainForName:kPreviousKeychainName
-                                                              clientID:kPreviousClientID
-                                                          clientSecret:kPreviousClientSecret];
-  if (authorization) {
+  GTMKeychainStore oldKeychainStore = [[GTMKeychainStore alloc] initWithItemName:kPreviousKeychainName];
+  authSession =
+      [oldKeychainStore retrieveAuthSessionInGTMOAuth2FormatWithClientID:kPreviousClientID
+                                                            clientSecret:kPreviousClientSecret
+                                                            error:&error];
+  if (authSession) {
     // Remove previously stored GTMOAuth2-formatted data.
-    [GTMOAuth2KeychainCompatibility removeAuthFromKeychainForName:kPreviousKeychainName];
+    [oldKeychainStore removeAuthSessionAndReturnError:&error];
```

Comment:
@mdmathias should this be `removeAuthSessionWithError:`?

Reply:
https://github.com/google/GTMAppAuth/pull/216